recipe/meta.yaml

```diff
@@ -26,31 +32,34 @@ build:
 requirements:
   build:
     - python  # [win]
+    - 7za  # [win]
+    - curl  # [win]
+    - msinttypes  # [win and py27]
     - perl 5.22.2.1
-    - freetype 2.7|2.7.*
-    - fontconfig 2.12.*
+    - freetype 2.7|2.7.*  # [unix]
+    - fontconfig 2.12.*  # [linux]
```

Comment:
Why not? Is it just using the system copy instead? That would be a problem as we use `fontconfig` as a dependency of many other packages on OS X.